server/game/gameSystems/AttackStepsSystem.ts

```diff
@@ -108,8 +108,9 @@ export class AttackStepsSystem<TContext extends AbilityContext = AbilityContext>
             return false; // cannot attack yourself or your controller's cards
         }
         if (
-            targetCard.hasRestriction(AbilityRestriction.BeAttacked, context) ||
-            properties.attacker.effectsPreventAttack(targetCard)
+            ((targetCard.hasRestriction(AbilityRestriction.BeAttacked, context) &&
+              !targetCard.hasSomeKeyword(KeywordName.Sentinel)) || // sentinel keyword overrides "can't be attacked" abilities (SWU Comp Rules 2.0 7.5.11.D)
```

Comment:
nit: can we put the && clause in parens on one line to make it clearer the two conditions are together